Framework/PythonInterface/plugins/algorithms/SaveP2D.py

```diff
@@ -0,0 +1,195 @@
+import numpy as np
+import math
+from scipy.special import lambertw
+from mantid.kernel import *
+from mantid.api import *
+
+
+class SaveP2D(mantid.api.PythonAlgorithm):
```

Comment:
Upon changing the headers we can make this
```suggestion
class SaveP2D(PythonAlgorithm):
```